README.md

```
@@ -2,117 +1,342 @@
+- Click on `Run Workflow`
+- This will now run the workflow on your private branch and the diawi apk link and ipa links will be shared on slack
+
+<br/>
+
+## 🙏 Further Help
+
+For access to environment files or repository write permissions, contact the mobile app team.
+
+For additional documentation, refer to the [Ionic Framework Docs](https://ionicframework.com/docs).
+
+<br/>
```

Markdownlint (0.35.0)

Element: br
Inline HTML

(MD033, no-inline-html)